eth/_utils/db.py

```diff
@@ -35,14 +34,13 @@ def get_block_header_by_hash(block_hash: Hash32, db: 'BaseChainDB') -> BlockHead
     return db.get_block_header_by_hash(block_hash)
 
 
-def apply_state_dict(account_db: BaseAccountDB, state_dict: AccountState) -> BaseAccountDB:
+def apply_state_dict(state: BaseState, state_dict: AccountState) -> None:
+    account_db = state.account_db
```

Comment:
`eth/tools/fixtures/fillers/_utils.py::calc_state_root` also calls this function but hasn't been updated to use the new signature.